server/utils/placementRequests/checkYourAnswersUtils.ts

```diff
@@ -123,34 +123,30 @@ export const attachDocumentsSummaryListItems = (
   pageName: string,
   showActions: boolean,
 ) => {
-  const items: Array<SummaryListItem> = []
+  const personCrn: string = application.person.crn
```

Comment:
This now seems very similar to `attachDocumentsSummaryListItems` in the summaryListUtils above -- is this actual duplication?

Reply:
The entire files were clearly copied and pasted at some stage so yes, there is lots of duplication. I merged the core of the summary list generation to `getDocumentSummaryListItems` inside summaryListUtils so that both `attachDocumentsSummaryListItems` here and `attachDocumentsSummaryListItems` in the application journey can share it. 
Maybe there is scope to move more code into the shared routines - but document lists are coming from different entity types so pushing it much further will probably hit type issues.
Actually, using the fn that absorbs exceptions, these two fns become very small.